lib/foreman/renderer.rb

```diff
@@ -8,7 +8,7 @@ module Renderer
 
     ALLOWED_VARIABLES ||= [:arch, :host, :osver, :mediapath, :mediaserver, :static,
                            :repos, :dynamic, :kernel, :initrd,
-                           :preseed_server, :preseed_path, :provisioning_type]
+                           :preseed_server, :preseed_path, :provisioning_type, :bootcfg]
```

Comment:
I don't think this is needed, there's no `bootcfg` variable defined as `pxe_type` is kickstart, so it uses kickstart_attributes (only initrd/kernel).